Automattic/jetpack
# Reading Settings: Add Reading link to the Settings menu in Calypso for Jetpack sites

Fixes https://github.com/Automattic/wp-calypso/issues/72628

## Proposed changes:

* Add the Reading link to the Calypso menu for Jetpack self-hosted sites. 
* The Reading link is placed right below the Writing link - the same order as we have in WP Admin.

| Before | After |
| ------------- | ------------- |
| ![Markup on 2023-01-27 at 14:25:07](https://user-images.githubusercontent.com/25105483/215098017-6e9f93e5-3e61-4f08-ad4b-63295c39a944.png) | ![Markup on 2023-01-27 at 14:32:01](https://user-images.githubusercontent.com/25105483/215099059-326d6e2b-0449-46ef-a893-512e44baa888.png) |

### Other information:


## Jetpack product discussion


- related Slack discussion: p1674817763288049/1674685814.308319-slack-C02TCEHP3HA

## Does this pull request change what data or activity we track or use?
No, it does not.

## Testing instructions:
1. Apply this PR to your sandbox and sandbox `public-api.wordpress.com`.
2. Apply this PR to your Jetpack site (Jurassic NInja can be used). The following instructions may be helpful: PCYsg-eg0-p2.
3. Navigate to Calypso at https://wordpress.com and select the Jetpack site.
4. The Reading settings link should display and should lead to the Calypso Reading settings page.
